diff --git a/include/ldastoolsal/fstream.hh b/include/ldastoolsal/fstream.hh
index 0bdc4d2..8030c24 100644
--- a/include/ldastoolsal/fstream.hh
+++ b/include/ldastoolsal/fstream.hh
@@ -45,9 +45,6 @@ namespace LDASTools
         //// \cond EXCLUDE
         template < class _Traits >
         struct _Eq_traits
-            : public std::binary_function< typename _Traits::char_type,
-                                           typename _Traits::char_type,
-                                           bool >
         {
             bool
             operator( )( const typename _Traits::char_type& __x,
